audit-log-explorer/README.md

```diff
@@ -0,0 +1,78 @@
+# Importable Audit Log Explorer App
```

Comment:
I see a warning in the app about not running large queries due to the potential impact on other users of the instance. I think we should call that out here prominently since users may not click in to see those instructions in the app.